### DISCUSSION TITLE
Possible to change lettertype size?

### DISCUSSION THREAD

**Discussion by DeeDeeRanged** [keepassxreboot/keepassxc#10241] [Q&A]
Is it possible to change the font size for KeePassXC. I am running Debian testing/Trixie and the fontsize of KeePassXC is now to large.

**Comment by droidmonkey** [comment]
It's on the feature request list. However, in your case you likely need to adjust your Qt environment variables. See https://keepassxc.org/docs/KeePassXC_UserGuide#_command_line_options
